src/Main.cpp

```diff
@@ -1,49 +1,60 @@
+#include <genset_whisperpower_ddc/ControlCommand.hpp>
+#include <genset_whisperpower_ddc/VariableSpeed.hpp>
+#include <genset_whisperpower_ddc/VariableSpeedMaster.hpp>
 #include <iostream>
```

Comment:
I completely forgot that this existed. Just move most of the read_frame code into a separate function and add a "speed-info" subcommand that would display only the time and rotation speed.

Reply:
done.